src/com/intellij/idea/plugin/hybris/runConfigurations/LocalSapCXRunSettingsEditor.kt

```diff
@@ -0,0 +1,147 @@
+        addDebugPortFragment()
+        addDebugLineFragment(runConfiguration.getRemoteConnetion().launchCommandLine)
+        add(LogsGroupFragment())
+        addEnvironmentFragment()
+    }
+    private fun SettingsEditorFragmentContainer<LocalSapCXRunConfiguration>.addEnvironmentFragment() =
+        addEnvironmentFragment(
+            object : LabeledSettingsFragmentInfo {
+                override val editorLabel: String = ExecutionBundle.message("environment.variables.component.title")
+                override val settingsId: String = "maven.environment.variables.fragment"
+                override val settingsName: String = ExecutionBundle.message("environment.variables.fragment.name")
+                override val settingsGroup: String = MavenConfigurableBundle.message("maven.run.configuration.runner.options.group")
+                override val settingsHint: String = ExecutionBundle.message("environment.variables.fragment.hint")
+                override val settingsActionHint: String = ExecutionBundle.message("set.custom.environment.variables.for.the.process")
+            },
+            { sapCXOptions.environmentProperties },
+            { sapCXOptions.environmentProperties = it as MutableMap<String, String> },
+            { sapCXOptions.isPassParentEnv },
+            { sapCXOptions.isPassParentEnv = it },
+            hideWhenEmpty = true
+        )
+    private fun SettingsEditorFragmentContainer<LocalSapCXRunConfiguration>.addDebugHostFragment() = addLabeledSettingsEditorFragment(
+        object : LabeledSettingsFragmentInfo {
+            override val editorLabel: String = HybrisI18NBundleUtils.message("hybris.project.run.configuration.localserver.host.field")
+
+            override val settingsId: String = "hybris.localserver.debug.host.fragment"
+            override val settingsName: String = "remoteDebugHost"
+            override val settingsGroup: String? = null
+            override val settingsHint: String = ""
+            override val settingsActionHint: String? = null
+        },
+        { HostField(HybrisConstants.REMOTE_DEBUG_DEFAULT_HOST) },
+        { it, c -> c.host = it.sapCXOptions.remoteDebugHost ?: "" },
+        { it, c -> it.sapCXOptions.remoteDebugHost = c.host }
+    )
+
+
+    private fun SettingsEditorFragmentContainer<LocalSapCXRunConfiguration>.addDebugPortFragment() = addLabeledSettingsEditorFragment(
+        object : LabeledSettingsFragmentInfo {
+            override val editorLabel: String = HybrisI18NBundleUtils.message("hybris.project.run.configuration.localserver.port.field")
+
+            override val settingsId: String = "hybris.localserver.debug.port.fragment"
+            override val settingsName: String = "remoteDebugPort"
+            override val settingsGroup: String? = null
+            override val settingsHint: String = ""
+            override val settingsActionHint: String? = null
+        },
+        { PortField(HybrisConstants.REMOTE_DEBUG_DEFAULT_PORT) },
```

Comment:
it looks like hardcode. please consider more advanced configuration / identification in the next PR

Reply:
This is only to provide Default Values and placeholders. The Value is still changable. See Image in the First PR Comment